pac4j-core/src/main/java/org/pac4j/core/io/FileSystemResource.java

```diff
@@ -0,0 +1,40 @@
+package org.pac4j.core.io;
+
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+public class FileSystemResource implements WritableResource {
+
+	private String path;
+
+	public FileSystemResource(String path) {
+		this.path = path;
+	}
+
+	@Override
+	public boolean exists() {
+		// TODO Auto-generated method stub

```

Comment:
I'd prefer not to keep any `TODO`. My feeling is that either you throw some exception to say it is not implemented or you implement the real logic.